pairModule/pairingProcesses.py

```diff
@@ -25,58 +26,114 @@ def pairAllTeams():
 
 def pairTeam(team):
     people = getPeople(team)
-    previousPairSets = getPreviousPairs(team)
+    previousPairObjs = getPreviousPairObjs(team)
 
-    wfh_answers = inquirer.prompt([
+    lastPairing = previousPairObjs[0] if len(previousPairObjs) else None
+
+    if lastPairing:
+        print(f'\nLast pairing name: "{lastPairing["name"]}"\n')
+        printPairings(lastPairing["pairings"],
+                      [*lastPairing["unpaired"], *lastPairing["excluded"]])
+
+        redo_answer = inquirer.prompt([
+            inquirer.Confirm(
+                'redo',
+                message='Would you like to ignore and redo the last pairing (displayed above)?',
+                default=False
+            )
+        ])['redo']
+
+        print(
+            "\nThe last pairing will be used to set defaults for the upcoming questions.\n")
+    else:
+        redo_answer = False
+        print("\nNo previous pairings found.\n")
+
+    if lastPairing:
+        preselected_wfh_people = []
+        unpreselected_wfh_people = []
+        for person in people:
+            if lastPairing["location"].get(person, None) == "home":
```

Comment:
Another question for my own sake
What does the None in the .get mean?

Reply:
it is the default if person is not in that dictionary